demos/python_demos/object_detection_demo_ssd_async/object_detection_demo_ssd_async.py

```diff
@@ -31,147 +35,335 @@
 def build_argparser():
     parser = ArgumentParser(add_help=False)
     args = parser.add_argument_group('Options')
-    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
-    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model.",
+    args.add_argument('-h', '--help', help='Show this help message and exit.', action='help', default=SUPPRESS)
+    args.add_argument('-m', '--model', help='Required. Path to an .xml file with a trained model.',
                       required=True, type=str)
-    args.add_argument("-i", "--input",
-                      help="Required. Path to video file or image. 'cam' for capturing video stream from camera",
+    args.add_argument('-i', '--input', help='Required. Path to image or video file or index of camera.',
                       required=True, type=str)
-    args.add_argument("-l", "--cpu_extension",
-                      help="Optional. Required for CPU custom layers. Absolute path to a shared library with the "
-                           "kernels implementations.", type=str, default=None)
-    args.add_argument("-d", "--device",
-                      help="Optional. Specify the target device to infer on; CPU, GPU, FPGA, HDDL or MYRIAD is "
-                           "acceptable. The demo will look for a suitable plugin for device specified. "
-                           "Default value is CPU", default="CPU", type=str)
-    args.add_argument("--labels", help="Optional. Path to labels mapping file", default=None, type=str)
-    args.add_argument("-pt", "--prob_threshold", help="Optional. Probability threshold for detections filtering",
-                      default=0.5, type=float)
-    args.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
-    args.add_argument("-u", "--utilization_monitors", default='', type=str,
-                      help="Optional. List of monitors to show initially.")
+    args.add_argument('-l', '--cpu_extension',
```

Comment:
doubt if we still mention cpu_extension library for 2020.4 release.

Reply:
Removed it.